packages/firestore/lib/index.d.ts

```diff
@@ -494,12 +512,12 @@ export namespace FirebaseFirestoreTypes {
      * ```js
      * const user = await firebase.firestore().doc('users/alovelace').get();
      *
-     * console.log('Address ZIP Code', user.get('address.zip'));
+     * console.log('Address ZIP Code', user.get<string>('address.zip'));
```

Comment:
you wouldn't put `<string>` in the actual code.

Reply:
To clarify, it presumes the developer is using Typescript.

Reply:
I forgot that we're not using Typescript in any of the examples, good one!